data/maps/maps.asm

```diff
@@ -1,8 +1,14 @@
 map: MACRO
-	; label, tileset, permission, location, music, phone service flag, time of day, fishing group
-\1_MapHeader:
-	db BANK(\1_SecondMapHeader), \2, \3
-	dw \1_SecondMapHeader
+;\1: map name: for the MapAttributes pointer (see data/maps/attributes.asm)
+;\2: tileset: a TILESET_* constant
+;\3: environment: TOWN, ROUTE, INDOOR, CAVE, ENVIRONMENT_5, GATE, or DUNGEON
```

Comment:
Can you please go ahead and rename `ENVIRONMENT_5` to `ENCLOSED`? It was called `SHIP` in the original source, and behaves for day-night "outdoor" environments that you can't Fly/Dig/Teleport/Escape from. A shipdeck would work; I use it for rooftops as well.

Reply:
(Noted for a future PR.)